### PR TITLE
[WIP] Add a data-only dump to the tree for feedback

### PR DESCRIPTION
**Pull request by diogomatsubara**
Add initial commit with a data-only dump of demo dispatch

### CODE REVIEW

**Comment by kevgliss** [comment]
Thanks for this, wondering where we should store this? I'm inclined to put this in the https://github.com/Netflix/dispatch-docker repo instead of the main application repo (with a docs on how to restore). 

What do you think?

**Comment by diogomatsubara** [comment]
For me would be more useful to have it in this repo as I'm not using the dispatch-docker repo. I adapted the build configuration to be able to deploy on Platform.sh. I was thinking maybe I could add an additional `database` command to load sample data. That way for people trying they can just run `dispatch database load_sample_data` or something like that and they have an environment somewhat useful to play with. Would that be a good solution?

**Comment by kevgliss** [comment]
I think that's fine, but I would at least want to create a `data` directory for it.

File: src/dispatch/cli.py [hunk 1]
```diff
@@ -568,6 +568,10 @@ def dump_database():
         _env={"PGPASSWORD": password},
     )
 
+@dispatch_database.command("load_sample_data")
+def load_sample_data():
+    """Load sample data into the database."""
+    pass
```
Comment (kevgliss) [inline src/dispatch/cli.py]: Should we reuse the `restore` command/function with a different path?

Comment (diogomatsubara) [reply src/dispatch/cli.py]: I made changes to re-use the `restore` command. My sample data is still not working and I think I'll have to make some edits to it. I'll ping you here once I have something working. Thanks for the feedback!

**Comment by kevgliss** [comment]
I'm going to merge this in as I think it will be helpful to lots of folks just trying to experiment with dispatch.

**Comment by sakshigupta3** [comment]
I don't see any such option, "dispatch database load_sample_data" to load sample data. Was the design changed or could you please add a section for this in https://hawkins.gitbook.io/dispatch/cli

**Pull request merged**